src/bulls_and_cows_lib/game_options.cpp

```diff
@@ -4,5 +4,17 @@
 namespace bulls_and_cows {
 
     // TODO: define the body of the functions declared in game_options.cpp
+    // Ask the user to select an option of the menu
+    GameOptionsMenuChoice ask_game_options_menu_choice(std::istream& input_stream)
+    {
```

Comment:
Here you must ask the player which kind of modification of the options he wants to do